gemelli/base.py

```diff
@@ -21,25 +21,20 @@ def fit(self):
         should be implemetned by sub-method"""
 
     def transform(self):
```

Comment:
Might be helpful to copy these lines here:
https://github.com/cameronmartino/gemelli/blob/2b8edb591454d515cfdf123fce3dea3447d377ab/gemelli/factorization.py#L62-L72